resource/authentication_manager.go

```diff
@@ -37,66 +37,79 @@ func (a *AuthenticationManager) GetCredentialPath(key string) string {
 
 // Create a new container authentication credential and write it into the Agent's host file system. The credential file
 // live in a directory named by the key input parameter.
-func (a *AuthenticationManager) CreateCredential(key string, id string, ver string) error {
+func (a *AuthenticationManager) CreateCredential(key string, id string, ver string, isDev bool) error {
 	cred, err := GenerateNewCredential(id, ver)
 	if err != nil {
 		return errors.New("unable to generate new authentication token")
 	}
 
+	fileName := path.Join(a.GetCredentialPath(key), config.HZN_FSS_AUTH_FILE)
+	// The following strategy with group permission does not apply to "hzn dev"
```

Comment:
Please update this comment to describe what's going on more generically. Say something like this: For insecure creds, the group is not needed because the file permissions will be universal.